### DISCUSSION TITLE
Confusing info in README

### DISCUSSION THREAD

**Discussion by antim0118** [jwallet/spy-spotify#406] [General]
README.MD:
`Spytify records the same quality that Spotify outputs (Spotify Free 160kbps, Spotify Premium 320kbps), so the recorded copy will be indistinguishable from Spotify’s one.`

The problem is that you cannot get the same quality if you convert lossy to lossy format (*160kbps->160kbps* or *320kbps->320kbps* / *spotify's ogg->mp3*). You can preserve quality if you record *160kbps->wav* or *320kbps->wav*, but this method will take a lot of memory on the pc.

Some articles:
https://hydrogenaud.io/index.php?topic=55703.0
https://hydrogenaud.io/index.php?topic=109382.0

**Comment by ReXulEc** [comment]
yeah, i agree.
